zipkin-common/src/test/scala/com/twitter/zipkin/storage/DependencyStoreSpec.scala

```diff
@@ -160,20 +160,20 @@ abstract class DependencyStoreSpec extends JUnitSuite with Matchers {
   @Test def dependencies_headlessTrace {
     processDependencies(List(trace(1), trace(2)))
 
-    result(store.getDependencies(today + 1000)) should be(Dependencies.toLinks(List(trace(1), trace(2))))
+    result(store.getDependencies(today + 1000)) should contain theSameElementsAs(dep.links)
```

Comment:
This change is linked to https://github.com/openzipkin/zipkin-dependencies-spark/pull/18